Gordon360/Controllers/ProfilesController.cs

```diff
@@ -476,14 +480,28 @@ public async Task<ActionResult<FacultyStaffProfileViewModel>> UpdateOfficeHours(
             return Ok(result);
         }
 
+        /// <summary>
+        /// Update mail location
+        /// </summary>
+        /// <param name="value">mail location</param>
+        /// <returns></returns>
+        [HttpPut]
+        [Route("mailstop")]
+        public async Task<ActionResult<FacultyStaffProfileViewModel>> UpdateMailStop([FromBody]string value)
+        {
+            var username = AuthUtils.GetUsername(User);
+            var result = await _profileService.UpdateMailStopAsync(username, value);
+            return Ok(result);
+        }
+
         /// <summary>
         /// Update privacy of mobile phone number
         /// </summary>
         /// <param name="value">Y or N</param>
         /// <returns></returns>
         [HttpPut]
         [Route("mobile_privacy/{value}")]
-        public async Task<ActionResult> UpdateMobilePrivacyAsync(string value)
+        public async Task<ActionResult> UpdateMobilePrivacyAsync([FromBody]string value)
```

Comment:
This is non-blocking, but I'm curious why this was changed. These attributes are optional, ASP.NET Core will still take the `value` argument from the body of the request without this attribute, as long as that is the first place where the argument exists in the request.

Reply:
I was not the one working with this route, but we have noticed we get an error message in some routes whenever we don't add [FromBody]. I am also not quite sure why. 

In another project, we had initially written the route to update the office hours without the from body and it seemed to be working, after a while another group tested and it would only work when adding this in the parameters.

Reply:
Hmm, interesting. As far as I am aware, it should not break anything to not have the attribute. I know that Swagger behaves differently (it will always put values in the URLSearchParams unless they are declared as belonging somewhere else like the body), but the API itself searches all locations for the argument when it isn't specified.

Regardless, this isn't bad the way it is.

Reply:
I think that might explain it.  My slightly fuzzy memory is consistent with them testing on Swagger and having this problem, and making this fix.  That would explain why it's only in a few places (where testing in Swagger happened).